galerautils/src/gu_asio.cpp

```diff
@@ -5,6 +5,7 @@
 #include "gu_config.hpp"
 #include "gu_asio.hpp"
 
+#include <boost/bind.hpp>
```

Comment:
Code uses ```boost::bind``` explicitly, so I think it is good to include ```boost/bind.hpp``` explicitly too and not to rely on other headers on this.